scripts/reports/day_report_cronjob.yaml

```diff
@@ -1,17 +1,24 @@
-apiVersion: batch/v1
+apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
   name: dayreport
 spec:
-  schedule: "0 0 * * *"
+  schedule: "0 0 * * *"  #  At 00:00
   jobTemplate:
     spec:
       template:
         spec:
           containers:
           - name: dayreport
-            image: busybox
+            image: docker.io/library/postgres:latest
```

Comment:
It's generally better to pin your image to a particular version (e.g., `postgres:12` or `postgres:13`). This avoids problems when `postgres:latest` is suddently referring to a major version upgrade.

Reply:
Sure we will make that change. Thanks for all your suggestion.